be/src/cloud/cloud_full_compaction.cpp

```diff
@@ -295,7 +295,14 @@ void CloudFullCompaction::do_lease() {
     }
 }
 
-Status CloudFullCompaction::_cloud_full_compaction_update_delete_bitmap(int64_t initiator) {
+Status CloudFullCompaction::_cloud_full_compaction_update_delete_bitmap_if_necessary(
+        int64_t initiator) {
+    if (_tablet->keys_type() != KeysType::UNIQUE_KEYS) {
```

Comment:
It would be more intuitive to put this judgment logic outside of the function call

Reply:
already change the code, take a look please @zhannngchen